src/index.js

```diff
@@ -134,6 +138,41 @@ class Mongo {
       throw new Error('User not found');
     }
   }
+
+  async createSession(userId: string, ip: string, userAgent: string): Promise<SessionType> {
+    const ret = await this.sessionCollection.insertOne({
+      userId,
```

Comment:
Uh I'm not sure. I don't think it matters since user id is  already guaranteed to be unique.